pinot-common/src/main/java/org/apache/pinot/common/metrics/ServerMeter.java

```diff
@@ -196,7 +196,12 @@ public enum ServerMeter implements AbstractMetrics.Meter {
   PREDOWNLOAD_FAILED("predownloadFailed", true),
 
   // reingestion metrics
-  SEGMENT_REINGESTION_FAILURE("segments", false);
+  SEGMENT_REINGESTION_FAILURE("segments", false),
+
+  /**
+   * Approximate heap bytes used by the mutable JSON index at the time of index close.
+   */
+  REALTIME_JSON_INDEX_MEMORY_USAGE("bytes", true);
```

Comment:
(minor) Consider renaming to match the class name
```suggestion
  MUTABLE_JSON_INDEX_MEMORY_USAGE("bytes", true);
```